lib/Common/Memory/HeapAllocator.cpp

```diff
@@ -146,7 +153,15 @@ void HeapAllocator::Free(void * buffer, size_t byteSize)
         return;
     }
 #endif
-    free(buffer);
+
+    if (CONFIG_FLAG(PrivateHeap))
+    {
+        HeapFree(GetPrivateHeap(), 0, buffer);
```

Comment:
I'll think about this for a future change, but I think this is something we want to get in early this week so I'd like to check in at first without making that change and re-evaluate with @curtisman